src/client/browserType.ts

```diff
@@ -32,13 +32,14 @@ import { assert, makeWaitForNextTask, mkdirIfNeeded } from '../utils/utils';
 import { SelectorsOwner, sharedSelectors } from './selectors';
 import { kBrowserClosedError } from '../utils/errors';
 import { Stream } from './stream';
+const packageVersion = require('../../package.json').version;
 
 export interface BrowserServerLauncher {
   launchServer(options?: LaunchServerOptions): Promise<BrowserServer>;
 }
 
 export interface BrowserServer {
-  process(): ChildProcess;
+  process(): ChildProcess | undefined;
```

Comment:
Please do not change this signature, it's not backwards-compatible. Instead, throw an error from `process()` when running in "connect over cdp" mode.